openstudiocore/src/model/BuildingUnit.cpp

```diff
@@ -80,6 +80,20 @@ namespace detail {
     return BuildingUnit::iddObjectType();
   }
 
+  std::vector<ModelObject> BuildingUnit_Impl::children() const {
+    std::vector<ModelObject> result;
+    if( boost::optional<RenderingColor> r = this->renderingColor() ) {
```

Comment:
@jmarrec on this one (and story, etc) can we be explicit and add a comment that we want this to only parent the colors?  i dont want someone to think we just forgot to parent the spaces and add that in later.

Reply:
Not 100% sure I follow here, BuildingUnit doesn't have any children aside from the rendering color, and I think the other that do have a rendering color already define their children properly.